SquareLine/boards/custom_waveshare_7inch/components/ws_7inch/include/bsp/ws_7inch.h

```diff
@@ -68,13 +68,13 @@ extern "C" {
  * @brief Init I2C driver
  *
  */
-void bsp_i2c_init(void);
+esp_err_t bsp_i2c_init(void);
 
 /**
  * @brief Deinit I2C driver and free its resources
  *
  */
-void bsp_i2c_deinit(void);
+esp_err_t bsp_i2c_deinit(void);
```

Comment:
Please, update the comment:
```
 *
 * @return
 *      - ESP_OK                On success
 *      - ESP_ERR_INVALID_ARG   I2C parameter error
```

Reply:
Should there be ESP_FAIL as well?

Reply:
There should be all error codes, which can be returned. Then yes please.